python/ray/dag/dag_node.py

```diff
@@ -153,6 +155,9 @@ def experimental_compile(
                 executions is beyond the DAG capacity, the new execution would
                 be blocked in the first place; therefore, this limit is only
                 enforced when it is smaller than the DAG capacity.
+            _nccl_group: The NCCL group to use for this DAG. If None, the DAG
```

Comment:
can you mention the default nccl group may incur gpu memory overhead?

Reply:
Sure, will do.